docs/release-notes/2.42.md

```diff
@@ -0,0 +1,122 @@
+
+- allocate resources efficiently by focusing on high-resolution data for key services
+- reduce storage requirements by adjusting resolution for less critical components
+- align your monitoring setup with the specific needs of your environment
+  
+This feature is currently accessible via the PMM API and will be integrated into the user interface in a future release.
+
+## Improved connection management for PostgreSQL Services and pmm-agent
+
+PMM now offers enhanced connection management capabilities for PostgreSQL services and the PMM Agent, to optimize resource utilization and prevent potential performance issues caused by excessive connections.
+
+### PostgreSQL services
+
+For PostgreSQL services, you can now set a maximum limit on the number of connections that the PostgreSQL exporter can open to the same PostgreSQL instance.
+
+This feature, previously available only through the CLI, is now also accessible through the PMM web interface as well:
+
+- PostgreSQL service: **PMM Configuration > Add Service > PostgreSQL > Configuring PostgreSQL Service**
+- RDS PostgreSQL service: **PMM Configuration > Add Service > Amazon RDS > Discover with Amazon Credentials > RDS PostgreSQL**
+
+![!image](../_images/Max_Connection_Limit.png)
+
+By setting a maximum connection limit, you can prevent excessive connections during concurrent operations, and ensure that connections are closed promptly to avoid idle connections.
+
+When adjusting the maximum number of connections, consider the following:
+
+- higher values might be needed for larger or busier instances.
+- setting the limit too high can impact performance.
+- if no limit is specified or the option is disabled, the server will manage the connection limits automatically.
+
+### PMM Agent
+
+Similarly, you can now limit the number of connections that the PMM Agent opens to monitored databases for QAN, Advisors, and other systems.
+
+Starting with this release, pmm-agent tracks all required resources (primarily connections), and opens no more than two connections per database instance simultaneously.
+
+You can change this default value by setting a parameter or environment variable when starting the pmm-agennt.
+
+## Experimental PMM Self-monitoring dashboard
+
+We've added a new experimental **PMM Health** dashboard to provide detailed insights into the health and performance of PMM itself.
+
+The dashboard reflects the PMM architecture and covers key components such as overall component status, node health, Query Analytics, PMM-managed, Grafana, VictoriaMetrics, ClickHouse, and PostgreSQL metrics.
+
+This initial version is available in the **Experimental** folder after updating PMM and contains a starter set of key metrics to help identify potential issues and ensure optimal operation.
+
+For more information about this new dashboard, see the [Keeping an eye on the eye blog post]().
```

Comment:
Unless intentionally, we are missing the url here :)

Reply:
@rnovikovP, could you please share the URL when you have it?